_schemaV2.graphql

```diff
@@ -10556,8 +10556,11 @@ type DismissTaskSuccess {
   task: Task!
 }
 
-# Display texts for the order based on its state
+# Display texts for the order based on its state and order shipping/payment states
 type DisplayTexts {
+  # First paragraph of the message text for the order
+  messageText(format: Format = MARKDOWN): String
```

Comment:
This might be late but I feel `message` and `title` are pretty self explanatory (and scoped under `DisplayTexts` already) and more concise, without the `Text` suffix.

Reply:
Totally open to rename it. Not late as it's barely used for now and good to do it before Eigen actually start using it.